### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Export Bacpac Settings_ColorContrast: Color Contrast ratio for 'Previous', 'Next' & 'Cancel' button is less than 4.5:1

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6802]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application  Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS References: MAS1.4.3, Colour contrast

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Apply 'Solarized Light' colour theme.
8. Right click an on prem database and select Data-tier application wizard from Context menu.
9. Select forth radio button.
10. Navigate till next button & hit enter.
11. Check the colour contrast ratio of 'Previous', 'Next' & 'Cancel' button.

**Actual:**
Colour Contrast ratio for 'Previous', 'Next' & 'Cancel' button is less than 4.5:1

**Expected:**
Colour Contrast ratio for 'Previous', 'Next' & 'Cancel' button should be greater than 4.5:1.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
User will not be able to see the control properly.

**MAS Reference:**
MAS 1.4.3-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
![image](https://user-images.githubusercontent.com/53556874/63266106-cf530a00-c2ac-11e9-8510-8e98e4e696e5.png)

**Comment by udeeshagautam** [comment]
On hold for themes

**Comment by udeeshagautam** [comment]
Closing this for the same reason as https://github.com/microsoft/azuredatastudio/issues/6781

Since this bug is not related to the default theme and we have the default theme button's contrast getting fixed... closing this bug. please reactivate if required.
